async-query-core/src/test/java/org/opensearch/sql/spark/asyncquery/AsyncQueryCoreIntegTest.java

```diff
@@ -522,10 +519,7 @@ private void givenValidDataSourceMetadataExist() {
                 .setName(DATASOURCE_NAME)
                 .setConnector(DataSourceType.S3GLUE)
                 .setProperties(
-                    ImmutableMap.<String, String>builder()
-                        .put(GLUE_INDEX_STORE_OPENSEARCH_URI, "https://open.search.cluster:9200/")
-                        .put(GLUE_INDEX_STORE_OPENSEARCH_AUTH, AuthenticationType.NOAUTH.getName())
-                        .build())
+                    ImmutableMap.<String, String>builder().put("PARAM_NAME", "PARAM_VALUE").build())
```

Comment:
What is the context here for removing data source dependency?

Reply:
We want to remove the dependency from async-query-core as much as possible so that the library wouldn't bring too much dependency to the consumer of this library.